src/molecules/Address.tsx

```diff
@@ -0,0 +1,21 @@
+import type { FunctionComponent } from 'react';
+
+import type { FlexProps } from '@atoms';
+import { Blockie, Body, Flex } from '@atoms';
+
+interface Props extends Omit<FlexProps, 'variant'> {
+  address: string;
+}
+
+export const Address: FunctionComponent<Props> = ({ address, ...props }) => (
+  <Flex variant="rowAlign" {...props}>
+    <Blockie address={address} width="30px" minWidth="30px" height="30px" mr="3" />
```

Comment:
`Blockie` should be optional.
The caller may also want to define the size.

Reply:
Are you saying add `showBlockie` and `blockieSize` or something like that?

Reply:
I believe the Address should just deal with the font family and address ellipse, so we have another component that combines an Address with a blockie.
In core the alignement of the blockie and the address are different since we also use a label.

Reply:
I created two components. Will add the component with labels later as it requires translation. It is also present in Signer.